src/elements/project-element.js

```diff
@@ -0,0 +1,95 @@
+import TextualElement from './textual-element';
+
+const DEFAULT_PROJECT_ELEMENT = {
+  label: 'Project',
+  key: '@project',
+  data_name: 'project',
+  disabled: false,
+  hidden: false,
+  choices: []
+};
+
+export default class ProjectElement extends TextualElement {
+  constructor(parent, attributes) {
+    attributes.type = 'ProjectField';
+
+    const attrs = Object.assign({}, DEFAULT_PROJECT_ELEMENT, attributes);
+
+    super(parent, attrs);
+
+    // this._projectFilter = null;
+    // this._choices = [];
+
+    // if (attrs.choices) {
+    //   for (const choice of attrs.choices) {
+    //     this._choices.push(choice);
+    //   }
+    // }
+
+    this._disabled = !!attrs.disabled;
+    this._hidden = !!attrs.hidden;
+  }
+
+  // get choices() {
```

Comment:
Can we remove these comments that look like they were pasted from the status element?

Reply:
I kept them because I didn't know if I'd want that code later when I try to add `SETPROJECTFILTER`. But I think you're right - it's not really necessary, especially since it's borrowed from the status element. If I need it later it's an easy C+P.